### PR TITLE
Added the sample of DB size sensor for MS SQL.

### PR DESCRIPTION
**Pull request by slydiman**
## Proposed change

Added the sample of DB size sensor for MS SQL.

Note this sample will work after fixing the issue home-assistant/core#42715
Currently MS SQL is not supported at all.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sql.markdown [hunk 1]
```diff
@@ -157,3 +157,20 @@ sensor:
         unit_of_measurement: 'MiB'
 ```
 {% endraw %}
+
+#### MS SQL
+
+Use the same db_url as for the `recorder` integration. Change `DB_NAME` to the name that you use as the database name, to ensure that your sensor will work properly. Be sure `username` has enough rights to access the sys tables.
```
Comment (frenck) [inline source/_integrations/sql.markdown]:
```suggestion
Use the same `db_url` as for the `recorder` integration. Change `DB_NAME` to the name that you use as the database name, to ensure that your sensor will work properly. Be sure `username` has enough rights to access the sys tables.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by slydiman** [comment]
What `awaits-parent` means? What's next?

**Comment by frenck** [comment]
It is marked against the `next` branch and link to the core repository. Mean it is awaiting code? That is what the next branch is for @slydiman. Is that incorrect @slydiman ?

**Comment by slydiman** [comment]
> It is marked against the `next` branch and link to the core repository. Mean it is awaiting code? That is what the next branch is for @slydiman. Is that incorrect @slydiman ?

I have no idea what it is awaiting for.
https://github.com/home-assistant/core/issues/42715 is closed.
https://github.com/home-assistant/core/pull/42778 is merged.

@dgomes (author of SQL sensor) asked me to add this myself
> I invite you to contribute directly :) it's easy just follow the link https://github.com/home-assistant/home-assistant.github.io/tree/current/source/_integrations/sql.markdown

But your approval mechanism is too complicated.
Please update it as needed to merge and close finally.
Thanks in advance.

**Comment by dgomes** [comment]
Yep, this should have been accepted long time ago, it was next, now it's current

**Comment by frenck** [comment]
home-assistant/core#42778 <- Is now for the first time referenced in your last comment. Meaning it now becomes apparent for the first time the parent has been merged already.

I'll see where I can merge it into.

**Pull request merged**